vnext/Microsoft.ReactNative/ReactNativeHost.cpp

```diff
@@ -21,6 +22,31 @@ ReactNativeHost::ReactNativeHost() noexcept : m_reactHost{Mso::React::MakeReactH
     OutputDebugStringA(str.c_str());
   });
 #endif
+  RegisterLifecycleEvents();
+}
+
+void ReactNativeHost::RegisterLifecycleEvents() {
+  m_lifecycleEventListeners = single_threaded_vector<ILifecycleEventListener>();
+
+  m_enteredBackgroundRevoker = Application::Current().EnteredBackground(
```

Comment:
>Application::Current() [](start = 31, length = 22)

I would prefer the ReactNativeHost to know nothing about the application. Especially, we must not reference the implicit global one like this.
Currently we have a lot of issues with XAML Islands caused by reliance on such global objects.

Reply:
I can probably just listen to the application events in native modules directly like @acoates-ms said. I'll do some tests...